### DISCUSSION TITLE
Rimma's notes from SIG calls - items for discussion on SIG calls and/or conversion to tasks/issues on the project board

### DISCUSSION THREAD

**Discussion by PaulaPaul** [finos/DEI-SIG#59] [General]
@Perelmuter - this is a great place to post your notes from SIG calls for open discussion (thank you for the suggestion @psmulovics!)

**Comment by Perelmuter** [comment]
Here are the FINOS minutes https://docs.google.com/document/d/1nXLXwH662LOgKGpbhy7IzGiTXI_dgxH5PvcQZ7-bTeQ/edit from the May 1st DEI call - they are not 'Rimma's notes' and capture key discussion pts/actions from the last call
